### ISSUE TITLE
fullpageApi and Next.js

### ISSUE THREAD

**Issue by farisT** [alvarotrigo/react-fullpage#67]
I am having an issue with my Next.js setup and fullpageApi. Everything is working fine only the fullpageApi does nothing when im trying to access it or move a section down.

```
import React from 'react';
import ReactFullpage from '@fullpage/react-fullpage/dist/react-fullpage-commonjs';

const opts = {
  licenseKey: 'OPEN-SOURCE-GPLV3-LICENSE',
};

const ProjectSlider = (props) => (
  <div className='slider'>
    <ReactFullpage
     {...opts}
      render={({ state, fullpageApi }) => {
        return (
          <ReactFullpage.Wrapper>
            {props.children &&
            <div className='section section--home '>
                {props.children}
                <button onClick={() => fullpageApi.moveSectionDown()}>Hello</button>
            </div>
            }
              <div className='section project--one'>
                  <span className='project__number'>1</span>
              </div>
              <div className='section project--two'>
                  <span className='project__number'>2</span>
              </div>
              <div className='section project--three'>
                  <span className='project__number'>3</span>
              </div>
            
          </ReactFullpage.Wrapper>
        );
      }}
    />
  </div>
  );

  export default ProjectSlider
```

**Comment by alvarotrigo** [comment]
Probably related with https://github.com/alvarotrigo/react-fullpage/issues/27.
It is being solved in the dev branch. Will get merged for the next release soon.

**Comment by alvarotrigo** [comment]
@farisT can you please verify if you still having the issue in the recently released version 0.1.3?

**Comment by cmswalker** [comment]
Sorry for the confusion over the SSR implementations. You can require the standard library now as it defaults to common JS. I have added some example repos to help

See: alvarotrigo/react-fullpage#server-side-rendering
